### PR TITLE
[strings] Updated category consistency checker and documented it

### PR DESCRIPTION
**Pull request by RicoElectrico**
Hopefully translators won't do it manually anymore :)

### CODE REVIEW

**Review by pastk — APPROVED** [review]
Its a good first step!

ATM the output is not very helpful though, e.g.
```
Inconsistent category "fuel"
	fi : "Polttoaine" is not matched by ["Huoltoasema"]
	ru : "Заправка" is not matched by ["бензин", "дизель", "топливо", "газ"]
```

Its not clear what is not consistent with what and where to fix it.
So the best would be to show corresponding files names, ideally with line numbers.

After that we can add this validation run into `generate_localization.sh` and github CI.

**Comment by RicoElectrico** [comment]
Well, only one name can be assigned in types_strings.txt as opposed to multiple names in categories.txt, but you're right, it's not foolproof enough. I would rather show string names than line numbers, though.

**Review by biodranik — APPROVED** [review]
Good start! Can you please add categories fixes too in a separate commit??

File: tools/ruby/category_consistency/omim_parsers.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 module OmimParsers
-  LANGUAGES = %w(en ru ar cs da nl fi fr de hu id it ja ko nb pl
-                 pt ro es sv th tr uk vi zh-Hans zh-Hant he sk)
+
+  # To update the list, run in root directory:
+  # sed -nEe "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/\1/p" "data/strings/strings.txt" | sort -u | tr '\n' ' ' | sed -e 's/,$//' | fold -s -w48; echo
+  LANGUAGES = %w(af ar be bg ca cs da de el en en-GB es es-MX et
```
Comment (biodranik) [inline tools/ruby/category_consistency/omim_parsers.rb:5]: Can new languages be detected automatically, and warned by the script?

**Pull request merged**